x/wasm/migrations/v3/legacy_types.go

```diff
@@ -0,0 +1,1219 @@
+package v3
```

Comment:
Good file name. It is important to preserve a copy of the original type for deserialization